docs/managing_deis/add_remove_host.rst

```diff
@@ -284,3 +284,29 @@ This can be achieved by making a request to the etcd API. See `remove machines`_
 .. _`remove machines`: https://coreos.com/docs/distributed-configuration/etcd-api/#remove-machines
 .. _`removing monitors`: http://ceph.com/docs/hammer/rados/operations/add-or-rm-mons/#removing-monitors
 .. _`removing OSDs`: http://docs.ceph.com/docs/hammer/rados/operations/add-or-rm-osds/#removing-osds-manual
+
+Graceful Shutdown
+~~~~~~~~~~~~~~~~~
+
+The contrib user-data provides a unit which provides some experimental logic to clean-up a Deis
+node's ceph and etcd membership before reboot, shutdown or halt events. Currently, the procedure will
+also be triggered if it is manually stopped via systemctl.
+
+The unit requires that the optional deis-store-admin component is installed.
+
+.. code-block:: console
+
+    root@deis-1:/# deisctl install deis-store-admin
+    root@deis-1:/# deisctl start deis-store-admin
+
+To enable the feature you must enable and start the unit:
+
+.. code-block:: console
+
+    root@deis-1:/# systemctl enable graceful-deis-shutdown
+    root@deis-1:/# systemctl start graceful-deis-shutdown
+
+The unit is now active and will now be stopped ahead of the store components, docker and etcd.
+The script (/opt/bin/graceful-shutdown.sh) will determine if the ceph cluster is healthy, and attempt
```

Comment:
/opt/bin/graceful-shutdown.sh should be wrapped in double-ticks, and Ceph should be capitalized in both places.